test/java/org/opendatakit/briefcase/util/InMemoryFormCache.java

```diff
@@ -0,0 +1,37 @@
+package org.opendatakit.briefcase.util;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.opendatakit.briefcase.model.BriefcaseFormDefinition;
+
+public class InMemoryFormCache implements FormCacheable {
+  private final Map<String, String> hashes = new HashMap<>();
```

Comment:
With map names, I like to give more information about the keys and values, especially since `<String, String>` tells us little. Here I would say `hashesByFormName` or something like that.